setup.py

```diff
@@ -10,13 +10,22 @@
 # SPDX-License-Identifier: ISC
 
 import setuptools
+import setuptools.command.build_py
+import os
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+class BuildMakeCommand(setuptools.command.build_py.build_py):
+  """Run make."""
+
+  def run(self):
+    os.system('make')
```

Comment:
If this just going to call out to something, please use CMake rather than make.  We are eventually going to target Windows, and CMake can generate shared libraries on Windows much easier than a Make system would be.

Reply:
ANTLR has first class support for CMake by the way: https://github.com/antlr/antlr4/blob/master/runtime/Cpp/cmake/Antlr4Package.md

Reply:
Thanks. I'm working on a CMakeLists.txt

Reply:
I've switched the build to CMake.